### DISCUSSION TITLE
Update Information - v.1.18.0

### DISCUSSION THREAD

**Discussion by pafuhana1213** [pafuhana1213/KawaiiPhysics#150] [Announcements - アナウンス]
C++のビルド環境を持っていない方は下記のバイナリ版をご利用ください
For those who do not have a build environment, please use the binary version provided below
Github : https://github.com/pafuhana1213/KawaiiPhysics/releases/tag/20241201-v1.18.0
Booth : https://pafuhana1213.booth.pm/items/5943534

## Add UE5.5 support!
遅くなってすみません！先日リリースしたUE5.5のサポートを追加しました！
Sorry for my late update... I've started to support UE5.5 now!

## Add copyright to codes
今更ですが、各コードにコピーライトを追加しました。
It's late, but I added copyright to each code.
`// KawaiiPhysics : Copyright (c) 2019-2024 pafuhana1213, MIT License`

## Fix build error in UE5.4
UE5.4にて発生していたパッケージ時のエラーを修正しました。
I've fixed any issue while packaging in UE5.4

**Comment by muchogrande** [comment]
Man, you update faster than many paid plugins - huge ありがとうございます for continuing to provide this excellent plugin!

**Comment by JadeHaru** [comment]
Thank you so much and Merry Christmas!
